keda/templates/12-keda-deployment.yaml

```
@@ -30,6 +30,9 @@ spec:
         {{- if .Values.podLabels.keda }}
         {{- toYaml .Values.podLabels.keda | nindent 8}}
         {{- end }}
+        {{- if .Values.podIdentity.azureWorkload.enabled }}
+        azure.workload.identity/use: {{ .Values.podIdentity.azureWorkload.enabled | quote }}
```

Nit - Either we hard-code it if we use an IF, or we always add the label based on the property.

My preference would be this:
```suggestion
        azure.workload.identity/use: "true"
```